src/Framework/Support/Facades/ActionScheduler/AsBackgroundJobsFacade.php

```diff
@@ -42,6 +42,25 @@ public function enqueueAsyncAction(
         }
     }
 
+    /**
+     * @unreleased
+     */
+    public function enqueueAction(
+        int $timestamp,
+        string $hook,
+        array $args,
+        string $group,
+        bool $unique = false,
+        int $priority = 10
+    ): int {
+        $enqueuedAction = $this->getActionByHookArgsGroup($hook, $args, $group, 'ids');
+        if (empty($enqueuedAction)) {
+            return as_schedule_single_action($timestamp, $hook, $args, $group, $unique, $priority);
+        }
+
+        return $enqueuedAction[0];
+    }
```

Comment:
Please provide a description for this. It looks like it conditionally enqueues a single action so long as that action isn't already queued? We may want to call this `safelyEnqueueAction` or something like that.

Reply:
I removed this method because I don't find this helper class very useful. Mainly because of the limited Action Scheduler API support and performance reasons. If we want to build an AS wrapper, there's a better and much more performant approach we can take (using the `ActionScheduler_Store` instance). Check the new `BatchMigrationRunner` controller.